src/model.ts

```diff
@@ -242,6 +242,26 @@ export class GitExtension implements IGitExtension {
     await this.refreshStatus();
   }
 
+  /**
+   * Match files status information based on a provided file path.
+   *
+   * If the file is tracked and has no changes, undefined will be returned
+   *
+   * @param path the file path relative to the server root
+   */
+  getFile(path: string): Git.IStatusFile {
+    const matchingFiles = this._status.files.filter(status => {
```

Comment:
Would it be more efficient to use find as only the first match is returned?

```suggestion
    const matchingFiles = this._status.files.find(status => {
```